### PR TITLE
improve nextjs and self-hosting docs

### PR DESCRIPTION
**Pull request by Vadman97**

### CODE REVIEW

File: docs-content/general/2_company/open-source/self-host-hobby.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ To run our app locally, head to our [github repo](https://github.com/highlight/h
 ## Docker Compose
 
 ```shell
+# ensure git submodules are cloned
+git submodule update --init --recursive
```
Comment (jay-khatri) [inline docs-content/general/2_company/open-source/self-host-hobby.md:16]: we should do this in the `readme` of the main repo as well (if you haven't already)

File: docs-content/general/3_getting-started/fullstack-frameworks/next-js/1_overview.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ If you want to monitor backend errors, this API wrapper will send your errors to
 ```typescript
 import { Highlight } from '@highlight-run/next'
 
-export const withHighlight = Highlight()
+export const withHighlight = Highlight({projectID: 'YOUR_PROJECT_ID'})
```
Comment (jay-khatri) [inline docs-content/general/3_getting-started/fullstack-frameworks/next-js/1_overview.md:70]: Is this also in the product somewhere? We'll need to update that too.

**Review by jay-khatri — APPROVED** [review]

File: docs-content/general/2_company/open-source/self-host-hobby.md (hunk 1 above)
Comment (Vadman97) [reply docs-content/general/2_company/open-source/self-host-hobby.md:16]: yup i will!

File: docs-content/general/3_getting-started/fullstack-frameworks/next-js/1_overview.md (hunk 2 above)
Comment (Vadman97) [reply docs-content/general/3_getting-started/fullstack-frameworks/next-js/1_overview.md:70]: found another reference, thanks. our dogfooding usage of this in the app is already updated

Comment (jay-khatri) [reply docs-content/general/3_getting-started/fullstack-frameworks/next-js/1_overview.md:70]: sorry, I meant on the `setup/` page in https://app.highlight.io.

Comment (Vadman97) [reply docs-content/general/3_getting-started/fullstack-frameworks/next-js/1_overview.md:70]: ah makes sense, will do

**Pull request merged**